src/Ampersand/Classes/ViewPoint.hs

```diff
@@ -45,7 +45,7 @@ rulesFromIdentity identity
        meaningEN = "Identity rule" <> ", following from identity "<>name identity
        meaningNL = "Identiteitsregel" <> ", volgend uit identiteit "<>name identity
        mkKeyRule expression =
-         Ru { rrnm   = "identity_" <> name identity
+         Ru { rrnm   = "IDENT " <> name identity
```

Comment:
This will produce an invalid name. Names shouldn't contain spaces. You seem to confuse syntax with the name itself.

Reply:
Hm, yes you are right. too bad. I'll withdraw this change